src/Sql/dbo/Stored Procedures/OrganizationUser_ReadByUserIds.sql

```diff
@@ -0,0 +1,18 @@
+CREATE PROCEDURE [dbo].[OrganizationUser_ReadByUserIds]
+    @UserIds AS [dbo].[GuidIdArray] READONLY
+AS
+BEGIN
+    SET NOCOUNT ON
+
+    IF (SELECT COUNT(1) FROM @UserIds) < 1
+    BEGIN
+        RETURN(-1)
+    END
+
+	SELECT
```

Comment:
Reduce indentation here to match the rest of the sproc

Reply:
This one was strange, as that line was indented as expected locally and git wasn't showing a change. I tabbed+commit, then untabbed+commited and that seemed to resolve. Should be fixed.